test/Pauser.t.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./TestSetup.sol";
+import "../src/Pauser.sol";
+
+contract PauserTest is TestSetup {
+    uint256 public constant INITIAL_DELAY = 1200; // 20 minutes
+    Pauser pauser;
+    IPausable[] initialPausables;
+
+    function setUp() public {
+        setUpTests();
+        
+        // deploy+initialize the pauser
+        Pauser pauserImplementation = new Pauser();
+        initialPausables.push(IPausable(address(liquifierInstance)));
+        initialPausables.push(IPausable(address(etherFiOracleInstance)));
+        
+        bytes memory initializerData = abi.encodeWithSelector(Pauser.initialize.selector, initialPausables, INITIAL_DELAY,  address(roleRegistry));
+        pauser = Pauser(address(new UUPSProxy(address(pauserImplementation), initializerData)));
+
+        // giving the pauser contract the permissions it needs to pause and unpause the contracts
+        vm.startPrank(owner);
+        liquifierInstance.updatePauser(address(pauser), true);
+        liquifierInstance.transferOwnership(address(pauser));
+        etherFiOracleInstance.updateAdmin(address(pauser), true);
+        auctionInstance.updateAdmin(address(pauser), true);
+
+        // setting contract roles
+        vm.startPrank(admin);
+        roleRegistry.grantRole(pauser.PROTOCOL_PAUSER(), alice);
+        roleRegistry.grantRole(pauser.PROTOCOL_UNPAUSER(), alice);
+        roleRegistry.grantRole(pauser.PAUSER_ADMIN(), alice);
+        vm.startPrank(alice);
+    }
+
+    function test_singlePause() public {
+        // pausing the `liquifier` and `etherFiOracle`
+        pauser.pauseSingle(IPausable(address(liquifierInstance)));
+        pauser.pauseSingle(IPausable(address(etherFiOracleInstance)));
+        assertTrue(liquifierInstance.paused());
```

Comment:
check if `etherFiOracleInstance` is paused too

Reply:
added ✅